<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoPage</class>
 <widget class="QWidget" name="VideoPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="videoGeneral">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Basic.Settings.General</string>
     </property>
     <layout class="QFormLayout" name="formLayout_15">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::FieldGrowthPolicy::AllNonFixedFieldsGrow</enum>
      </property>
      <property name="labelAlignment">
       <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="minimumSize">
         <size>
          <width>170</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Basic.Settings.Video.BaseResolution</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>baseResolution</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_29">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QComboBox" name="baseResolution">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="currentText">
           <string notr="true"/>
          </property>
          <property name="duplicatesEnabled">
           <bool>false</bool>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="baseAspect">
          <property name="text">
           <string>AspectRatio</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="outputResLabel">
        <property name="text">
         <string>Basic.Settings.Video.ScaledResolution</string>
        </property>
        <property name="buddy">
         <cstring>outputResolution</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="outputResLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QComboBox" name="outputResolution">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="currentText">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="scaledAspect">
          <property name="text">
           <string>AspectRatio</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Basic.Settings.Video.DownscaleFilter</string>
        </property>
        <property name="buddy">
         <cstring>downscaleFilter</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="downscaleFilter"/>
      </item>
      <item row="3" column="0">
       <widget class="QComboBox" name="fpsType">
        <property name="currentText">
         <string>Basic.Settings.Video.FPSCommon</string>
        </property>
        <item>
         <property name="text">
          <string>Basic.Settings.Video.FPSCommon</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Basic.Settings.Video.FPSInteger</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Basic.Settings.Video.FPSFraction</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QStackedWidget" name="fpsTypes">
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item alignment="Qt::AlignmentFlag::AlignTop">
           <widget class="QComboBox" name="fpsCommon">
            <property name="currentText">
             <string notr="true">10</string>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <item>
             <property name="text">
              <string notr="true">10</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">20</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>24 NTSC</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>25 PAL</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">29.97</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">30</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">48</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>50 PAL</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">59.94</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">60</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_3">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item alignment="Qt::AlignmentFlag::AlignTop">
           <widget class="QSpinBox" name="fpsInteger">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>120</number>
            </property>
            <property name="value">
             <number>30</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QFormLayout" name="formLayout_4">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::FieldGrowthPolicy::ExpandingFieldsGrow</enum>
          </property>
          <property name="labelAlignment">
           <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Basic.Settings.Video.Numerator</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="fpsNumerator">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1000000</number>
            </property>
            <property name="value">
             <number>30</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Basic.Settings.Video.Denominator</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="fpsDenominator">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1000000</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="4" column="0">
       <spacer name="horizontalSpacer_24">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>170</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_9">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>386</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="videoMsg">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="class" stdset="0">
      <string>text-danger</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
